optimum/onnxruntime/trainer.py

```diff
@@ -0,0 +1,1274 @@
+#  Copyright 2021 The HuggingFace Team. All rights reserved.
```

Comment:
I think there are a lot of parts that are not needed here (we have the same issue with the `IPUTrainer`), but that can wait for later PRs for sure, just wanted to point it out.

Reply:
Yeah, sure! There are some features that might not work together with onnxruntime, I tested deepspeed since it is also considered by Microsoft/onnxruntime, but the rest remains unsure. A clean-up should upon this.